dozer-ingestion/src/connectors/mod.rs

```diff
@@ -231,7 +231,9 @@ pub fn get_connector(connection: Connection) -> Result<Box<dyn Connector>, Conne
 
 pub fn get_connector_info_table(connection: &Connection) -> Option<Table> {
     match &connection.config {
-        Some(ConnectionConfig::Postgres(config)) => Some(config.convert_to_table()),
+        Some(ConnectionConfig::Postgres(config)) => {
```

Comment:
Propagate error instead of `unwrap`

Reply:
How are we propagate the error if the return type is not having Result? panic?

Reply:
Yes changing the return type to `Result`. That's what I wanted to say. Sorry didn't make it clear.